package.json

```diff
@@ -58,6 +58,22 @@
     "vitepress": "^0.15.6",
     "yorkie": "^2.0.0"
   },
+  "peerDependencies": {
```

Comment:
Did you mean for these to be in https://github.com/vitejs/vite/blob/632a50acd693f4ba79af7d22f1df14cf20e66538/packages/vite/package.json? It doesn't make much sense to have them at the monorepo root since that isn't installed by users

Reply:
Seems like an oversight on my part, yes! Thanks for noticing